scss/detail.scss

```
@@ -0,0 +1,34 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
+.titulo-libro{
+  color: black;
+  font-family: 'Roboto', sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 38px;
+}
+
+.titulo-genero{
```

Sorry for the nitpick, could you add a space between a class name and the { ?
```suggestion
.titulo-genero {
```